projects/storefrontlib/src/cms-components/cart/cart-page-layout-handler.ts

```diff
@@ -23,22 +23,25 @@ export class CartPageLayoutHandler implements PageLayoutHandler {
         slots$,
         this.activeCartService.getActive(),
         this.selectiveCartService.getCart(),
+        this.activeCartService.getLoading(),
```

Comment:
What was the problem with `isStable` method that it wasn't used here?

Reply:
isStable() wasn't indicating whether data was being loaded or not. It returns false when data is being loaded or no data has been loaded. We are unable to tell is there is incoming data.

Reply:
Now we display nothing while data is still loading instead of empty cart.